components/RichText.tsx

```diff
@@ -95,6 +96,9 @@ export const RichText = ({
       sbCta: (props) => (
         <SbCta blok={props} />
       ),
+      sbText: (props) => (
+        <SbText blok={props} />
+      ),
```

Comment:
Storyblok Text component can now be inserted into the WYSIWYG. This allow me to insert eg, a type-3 (modular type 3 non-heading text) into the content like this
![Screenshot 2023-11-30 at 3 21 56 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/77dd4855-c488-40bc-bb10-5309cf4b2638)